### ISSUE TITLE
Cannot find package 'devalue'

### ISSUE THREAD

**Issue by gerhardcit** [aprakasa/sveltekit-node-docker#2]
If you do this again today?

Do you get this error?
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'devalue' imported from /app/build/server/index.js

**Comment by aprakasa** [comment]
Yeah, lot of breaking change recently https://github.com/sveltejs/kit/issues/3176
